pipelines/internal/commands/run/run.go

```diff
@@ -385,12 +381,32 @@ func buildRequest(filename, project string) (*genomics.RunPipelineRequest, error
 		})
 	}
 
+	resources := &genomics.Resources{
+		ProjectId:      project,
+		VirtualMachine: vm,
+	}
+	if *regions != "" && *zones != "" {
+		return nil, errors.New("both zones and regions have been supplied")
+	}
+	if *regions != "" {
+		regions, err := expandPrefixes(project, listOf(*regions), listRegions)
+		if err != nil {
+			return nil, fmt.Errorf("expanding regions: %v", err)
+		}
+		resources.Regions = regions
+	} else {
+		if *zones == "" {
```

Comment:
Definitely don't need to expand in this case, see what I suggested above.

Reply:
good point, done.